src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/collection-select/main.js

```diff
@@ -134,7 +152,7 @@ define(function() {
                         skin: 'wide',
                         slides: [
                             {
-                                title: this.sandbox.translate('sulu.media.move.overlay-title'),
+                                title: this.options.title,
```

Comment:
Doesn't this have to be translated anymore? Is this maybe also a change from @marcelmoosbrugger ?